src/services/dashboards/dashboard-detail/modules/DashboardMoreMenu.vue

```diff
@@ -70,16 +57,16 @@ const state = reactive({
     moreMenuItems: computed(() => {
         const menuItems = cloneDeep(defaultMenuItems.value);
         if (state.homeDashboardId === props.dashboardId) {
+            menuItems[0].disabled = true;
             menuItems[1].disabled = true;
```

Comment:
I think this way is safer.
```suggestion
            menuItems.find(d => d.name === MENU.DELETE).disabled = true;
            menuItems.find(d => d.name === MENU.SET_HOME).disabled = true;
```